tests/test_ops/test_merge_cells.py

```diff
@@ -11,31 +11,31 @@
 
 
 def test_sum_cell():
-    inputs_x = torch.randn([2, 256, 32, 32])
+    inputs_x = torch.randn([2, 256, 14, 7])
```

Comment:
We should test `SumCell`  and concat `ConcatCell` with divisible and indivisible both